relayer/relayer/src/handlers/eth.rs

```diff
@@ -117,6 +123,33 @@ impl EthereumEventHandler {
     }
 }
 
+async fn past_request(
+    most: &MostInstance,
+    connection: &AzeroConnectionWithSigner,
+    committee_id: u128,
+) -> Result<bool, EthereumEventHandlerError> {
+    if most
+        .is_in_committee(
+            connection.as_connection(),
+            committee_id,
+            connection.account_id().clone(),
+        )
+        .await?
+    {
+        return Ok(false);
+    }
+
+    if committee_id
```

Comment:
Can this even happen? Relayer should not keep on running if it does

Reply:
It will crash when executing the request - it shouldn't happen if the rotation is performed on both ends of the bridge, only happens if governance makes a mistake

Reply:
I see - yet still best to fail early if you can